.changeset/tall-actors-call.md

```diff
@@ -0,0 +1,5 @@
+---
+'xstate': patch
+---
+
+Upgrade to TypeScript 5.4
```

Comment:
This changeset should be removed - our consumers are not that concerned with what TS version our monorepo is using.